docs/versioning-policy.md

```diff
@@ -0,0 +1,98 @@
+
+### Breaking changes/stability guarantees
+
+We consider the following changes as *breaking*, highlighting them in changelogs and tracking them in [the release overview](./releases.md).
+The short tags at the beginning are the table column headers in said release overview.
+
+- *config*: Changing the configuration file in a backwards-incompatible way.
+- *auth*: Changing the auth-system in a backwards-incompatible way.
+- *cli*: Changing the stabilized parts of the CLI in a backwards-incompatible way.
+- *meili*: Requiring a new version of MeiliSearch.
+- *pg*: Dropping support for a non-EOL version of PostgreSQL.
+- *tmod*: Requiring a newer version of the Tobira module in Opencast.
+- *ocversion*: Requiring a new Opencast version.
+- *ocreq*: Adding new Opencast requirements (e.g. having static-file authorization disabled).
+
+
+### Non-breaking changes
+
+All changes not listed in the previous section are considered *non-breaking* by us.
+We may release these changes at any time.
+And while most will likely be mentioned in the changelog somewhere, some may not.
+
+To name a few things explicitly (this is not a complete list!):
+
+- Changing the configuration file in a backwards-compatible way (e.g. adding new fields with default values, allowing more values for an existing field, ...).
+- Adding new non-required `x-tobira-*` headers to the auth system.
+- Dropping support for an [EOL-version of PostgreSQL](https://www.postgresql.org/support/versioning/) (e.g. PostgreSQL 10 after Nov 10, 2022)
+- Database schema changes: we always provide scripts to easily migrate the database.
+  Tobira automatically runs the appropriate scripts when starting, so you don't have to do anything.
+- Anything that requires a search index rebuild.
+  These rebuilds will be performed automatically after updating Tobira and should be done within a few seconds (for tens of thousands of events/series).
+- Any changes to the GraphQL API. **This API is internal and we will break it very regularly.**
+- Any changes to emitted HTML, DOM nodes, CSS, and JS.
+- Any user-facing changes, like adding features, removing features, changing features, change the design, ...
+
+
+<br>
+<br>
+
+---
+
+## Stabilized parts of Tobira
+
+See the section "Breaking and non-breaking changes" for context.
+
+### CLI (command line interface)
+
+- `tobira serve`: starts the web server.
+- `tobira worker`: runs forever syncing with Opencast and keeping the search index up to date.
+- `tobira check`: checks for some potential problems with Tobira's configuration and environment.
+  Will return a non-zero error code if some problem was found.
+- `tobira sync run [--daemon]`
+- `tobira search-index update [--daemon]`
+
+And the `-c config-file` option for all these commands.
+
+
+### The GraphQL API
+
+Nothing is stabilized, everything may change.
+
+
+### Emitted HTML, DOM-nodes, CSS, JS
+
+Nothing is stabilized, everything may change.
```

Comment:
You forgot to mention that we may change the readme page and the list of dependencies… :D
Again, don't start non-comprehensive lists of random other things just for the sake of adding more documentation.

```suggestion
```